iceoryx_binding_c/include/iceoryx_binding_c/publisher.h

```diff
@@ -40,7 +40,7 @@ typedef struct
     bool offerOnCreate;
 
     /// @brief describes whether a publisher blocks when subscriber queue is full
```

Comment:
Please adjust the comment.

Reply:
Why, the comment is stil true. It's only the data type that changed it's name, note the member